src/vs/workbench/api/browser/mainThreadWebview.ts

```diff
@@ -160,7 +160,7 @@ export class MainThreadWebviews extends Disposable implements MainThreadWebviews
 	public async $postMessage(handle: WebviewPanelHandle, message: any): Promise<boolean> {
 		const webview = this.getWebview(handle);
 		const editors = this._editorService.visibleControls
-			.filter(e => e instanceof WebviewEditor)
+			.filter(e => (e as WebviewEditor).isWebviewEditor)
```

Comment:
Many of our other editors also are used in `instanceof` expressions. For these cases, we could either add an import expression to import the editor, or add a unique property like `isWebviewEditor `